### DISCUSSION TITLE
HW3 part 2

### DISCUSSION THREAD

**Discussion by ElisDiaz46** [cooperrc/computational-mechanics#168] [Q&A]
def a_freefall(v,c=0.25,m=60):
    a=-c/m*v**2*np.sign(v)-9.81
    return a

I have this function for the tennis and lacrosse ball, I am trying to consider but the drag force and mass ranges of both of the balls in their respective functions. How do I consider the range mass of the tennis ball 56.0–59.4g?

**Comment by Maid-Unknown** [comment]
I thing just take the mean will be fine

**Comment by Maid-Unknown** [comment]
which is 57.7
